### ISSUE TITLE
Multiple subscriptions.

### ISSUE THREAD

**Issue by viacheslavsaloid** [ngneat/subscribe#3]
Hello

It's really cool lib. Thank you.

How can i provide multiple subscribes ? 

<img width="799" alt="Screenshot 2022-01-20 at 14 02 59" src="https://user-images.githubusercontent.com/70704363/150335270-9f03b5e8-7f5b-47b2-a3f7-30e66ee662d8.png">

I think, this way is not good enough. But cant figure how to make it better?

**Comment by NetanelBasal** [comment]
Use combineLatest?

**Comment by viacheslavsaloid** [comment]
I thought, that with combineLatest we will lost `let error` and `let loading`

because each one have a uniq data

**Comment by NetanelBasal** [comment]
Yes, but my thought was about sources from state management. You're welcome to create a PR that supports this feature.
